docs/LPC_Additional_LPC_Cycles.md

```diff
@@ -0,0 +1,248 @@
+## Why do we need to support additional LPC protocol cycles in "LPC Host"
+
+In the implementation of the LPC (LPC Host and LPC Peripheral) protocol we wrote
+earlier, only two types of LPC cycles were supported:
++ I / O cycles (Read and Write)
++ TPM cycles
```

Comment:
TPM cycles actually are not supported - those are basically the same as I/O cycles but have a different `START` field. Please see my comment below for details.

Reply:
These issues  were corrected by me.